OptionSelector/CQTableViewController.m

```diff
@@ -0,0 +1,153 @@
+//
+//  CQTableViewController.m
+//  Unit1WeekFourHomework
+//
+//  Created by Jackie Meggesto on 8/12/15.
+//  Copyright (c) 2015 Jackie Meggesto. All rights reserved.
+//
+
+#import "CQTableViewController.h"
+#import "CQCategory.h"
+#import "CQDetailTableViewController.h"
+
+
+
+@interface CQTableViewController ()
+@property (nonatomic) NSMutableArray *categories;
```

Comment:
It would be better to make this a non-mutable array property (just `NSArray`). You can still use a mutable array when initializing your data in `viewDidLoad`.